pegasus/test/test_i18n_hoc_routes.rb

```diff
@@ -13,6 +13,7 @@ def test_hoc_pages_for_all_locales
     header 'Host', 'hourofcode.com'
```

Comment:
There is subtle bug here. The bug was that changing the `rack-test` default host (from example.com to hourofcode.com) would undo HoC translations loaded into the I18n.back_end by the test. The fix is to simply change the host first, only then load the HoC translations.